pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -65,6 +66,7 @@ const (
 	adminKubeconfigKey          = "kubeconfig"
 	clusterVersionObjectName    = "version"
 	clusterVersionUnknown       = "undef"
+	clusterDeploymentGeneration = "hive.openshift.io/cluster-deployment-generation"
```

Comment:
rename the variable to indicate that it's used as an annotation (ie. `clusterDeploymentGenerationAnnotation`)